hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -849,7 +849,7 @@ objects:
           path: /host/var/log/auth.log
           sourceType: linux_audit
           whiteList: \.log$
-        - path: /host/var/lib/kubelet/pods/*/volumes/kubernetes.io~*/pvc-*/backplane-audit.log
+        - path: /host/var/lib/kubelet/pods/*/volumes/kubernetes.io~*/pvc-*/mount/backplane-audit.log
```

Comment:
Could you add this as a new path? Looking at the commercial env, the PR thttps://github.com/openshift/splunk-forwarder-operator/pull/174 was made to target only hive clusters